pom.xml

```diff
@@ -365,6 +365,7 @@ under the License.
                 <https.protocols>${https.protocols}</https.protocols>
               </properties>
               <showVersion>true</showVersion>
+<!--              <parallelThreads>1C</parallelThreads>-->
```

Comment:
I suppose this comment should be removed

Reply:
my local testing ... removed